packages/backend-core/src/events/events.ts

```diff
@@ -1,8 +1,12 @@
-import { Event } from "@budibase/types"
+import { Event, AuditedEventFriendlyName } from "@budibase/types"
 import { processors } from "./processors"
 import identification from "./identification"
 import * as backfill from "./backfill"
 
+export function isAudited(event: Event) {
```

Comment:
Can move this into `packages/backend-core/src/events/processors/AuditLogsProcessor.ts`